projects/kit/components/input-phone-international/input-phone-international.template.html

```diff
@@ -0,0 +1,74 @@
+<ng-container *tuiLet="options.metadata | async as phoneMetadata">
+    <div
+        *ngIf="countriesNames$ | async as countriesNames"
+        tuiGroup
```

Comment:
Do we need this wrapper? Maybe we can move all this to host directives to minimize unnecessary DOM nesting? We probably need to come up with some infrastructure for binding values to host directives, we can probably achieve it with signals and effects, I'll think about it, you can keep it as is for now.

Reply:
I have made that infrastructure, let's put a TODO here to get rid of nesting as much as possible.

Reply:
> let's put a TODO here to get rid of nesting as much as possible

Done in https://github.com/taiga-family/taiga-ui/pull/7580/commits/a4614994750d48b5ccf395520ac5f7a59838ce22